Sources/TuistCache/Mappers/Graph/FocusTargetsGraphMappers.swift

```diff
@@ -16,11 +16,18 @@ public final class FocusTargetsGraphMappers: GraphMapping {
         let graphTraverser = GraphTraverser(graph: graph)
         var graph = graph
         let includedTargets = includedTargets
-            .isEmpty ? Set(graphTraverser.allInternalTargets().map(\.target.name)) : includedTargets
+            .isEmpty ? Set(
+                graphTraverser.allInternalTargets()
+                    .union(graphTraverser.allExternalTestsTargets())
+                    .map(\.target.name)
+            ) : includedTargets
```

Comment:
What's the point of including the external test targets, but not their sources?

Reply:
Also, we shouldn't include ALL of them :D

Reply:
> What's the point of including the external test targets, but not their sources?
I'm not sure I understand. My intention was to include both the test targets and their sources. I can see that happening in generated projects. Could you please explaining what am I missing?

>Also, we shouldn't include ALL of them :D
Added a filter but do you think I should add `testsFromPackages` parameter to the function as well? I was just relying on graph that was created during `tuist fetch` and trusted that any package that still have tests target at this point is the one that is opted-in

Reply:
> Added a filter but do you think I should add testsFromPackages parameter to the function as well? I was just relying on graph that was created during tuist fetch and trusted that any package that still have tests target at this point is the one that is opted-in

Makese sense, didn't thought about it 👌